helm/flowforge/README.md

```diff
@@ -215,3 +215,83 @@ readinessProbe:
   successThreshold: 1
   failureThreshold: 3
 ```
+
+## Upgrading Chart
+
+### Unreleased
```

Comment:
```suggestion
### To 2.1.0
```

Reply:
As this will go live next week this makes sense. I will merge and then do the core docs merge, but not backport so it will go live with the 2.1.0 release